plugin/buycourses/view/sales_report.tpl

```diff
@@ -1,3 +1,16 @@
+<link rel="stylesheet" type="text/css" href="../resources/css/style.css"/>
+<ul class="nav nav-tabs buy-courses-sessions-tabs" role="tablist">
+    <li id="buy-courses-sessions-tab" class="{{ showing_courses_sessions ? 'active' : '' }}" role="presentation">
+        <a href="sales_report.php" aria-controls="buy-courses_sessions" role="tab">{{ 'Courses'|get_lang }} & {{ 'Sessions'|get_lang }}</a>

```

Comment:
The & is not a valid sign in many non-western languages. Please update to a 'CoursesAndSessions' language variable